libs/constants/index.js

```diff
@@ -61,10 +61,39 @@ const ACTIVE_LOCALES = new Set([
   "zh-tw",
 ]);
 
+const CSP_DIRECTIVES = [
+  "default-src 'self'",
+  "script-src " +
+    [
+      "'report-sample'",
+      "'self'",
+      "https://cdn.speedcurve.com/js/lux.js",
+      "'sha256-q7cJjDqNO2e1L5UltvJ1LhvnYN7yJXgGO7b6h9xkL1o='", // LUX
+      "https://www.google-analytics.com/analytics.js",
+      "https://www.google-analytics.com/analytics_debug.js",
+      "'sha256-JEt9Nmc3BP88wxuTZm9aKNu87vEgGmKW1zzy/vb1KPs='", // polyfill check
+      "https://polyfill.io/v3/polyfill.min.js",
+    ].join(" "),
+  "style-src 'report-sample' 'self' 'unsafe-inline'",
+  "object-src 'none'",
+  "base-uri 'self'",
+  "connect-src 'self'",
+  "font-src 'self'",
+  "frame-src 'self' https://interactive-examples.mdn.mozilla.net https://mdn.github.io https://yari-demos.prod.mdn.mozit.cloud https://mdn.mozillademos.org https://yari-demos.stage.mdn.mozit.cloud",
+  "img-src 'self' https://*.githubusercontent.com https://*.googleusercontent.com https://lux.speedcurve.com",
+  "manifest-src 'self'",
+  "media-src 'self'",
+  "worker-src 'none'",
+  "report-uri https://developer.mozilla.org/csp-violation-capture",
```

Comment:
@Gregoor Per our discussion over video, let's change this to a relative URL, so we send the dev/stage reports to our stage sentry project, and prod reports to our prod sentry project.
```suggestion
  "report-uri /csp-violation-capture",
```

Reply:
Per https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/report-uri, I think we should also add a `report-to` directive so we're ready for the future 😄.

```js
  "report-to /csp-violation-capture",
```

Reply:
No, on further thought, I don't think we should worry about `report-to` yet.

Reply:
@Gregoor Since we've already talked about it, I'm going to merge the [suggested change above](https://github.com/mdn/yari/pull/3583#discussion_r627687418) so we can test the CSP reporting on stage.